src/tests/Common/Coreclr.TestWrapper/CoreclrTestWrapperLib.cs

```diff
@@ -675,10 +678,10 @@ public int RunTest(string executable, string outputFile, string errorFile, strin
 
             // If a timeout was given to us by an environment variable, use it instead of the default
             // timeout.
-            string environmentVar = Environment.GetEnvironmentVariable(TIMEOUT_ENVIRONMENT_VAR);
+            string? environmentVar = Environment.GetEnvironmentVariable(TIMEOUT_ENVIRONMENT_VAR);
             int timeout = environmentVar != null ? int.Parse(environmentVar) : DEFAULT_TIMEOUT_MS;
             bool collectCrashDumps = Environment.GetEnvironmentVariable(COLLECT_DUMPS_ENVIRONMENT_VAR) != null;
-            string crashDumpFolder = Environment.GetEnvironmentVariable(CRASH_DUMP_FOLDER_ENVIRONMENT_VAR);
+            string? crashDumpFolder = Environment.GetEnvironmentVariable(CRASH_DUMP_FOLDER_ENVIRONMENT_VAR);
```

Comment:
For these two env vars and the ones in the other files, should we throw if they're not found?